### DISCUSSION TITLE
Google Cloud Storage connector configurations

### DISCUSSION THREAD

**Discussion by johnjanakiraman-dp** [apache/gluten#7738] [Q&A]
@srinivasst @surnaik I heard you would be able to help with GCS-specific configurations? Could you share them?

**Comment by surnaik** [comment]
Hey John, could you please let me know what configs are you exactly looking for? 

Currently, from the perspective of gluten, the only supported GCS configuration are these:
https://github.com/apache/incubator-gluten/blob/40fc52abc1d1f50bae2655a4fdc24222451b0fbe/shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala#L529

**Reply by johnjanakiraman-dp** [reply]
Thanks, @surnaik. Are there any additional configs for Velox and/or for performance tuning?

**Reply by surnaik** [reply]
Currently there's no configuration available from performance tuning perspective, the connector implementation is very basic as of now with only authentication configuration. We do have plans to make it better but as of now, these are the only configs available. Thanks!
